tiled/utils.py

```diff
@@ -733,7 +735,8 @@ def path_from_uri(uri) -> Path:
         path = Path(parsed.path[1:])
     else:
         raise ValueError(
-            "Supported schemes are 'file', 'sqlite', and 'duckdb'. "
```

Comment:
No change to this function. Now, unlike when we started this PR, `path_from_uri` is only ever called on local filesystem paths.